src/Command/Environment/EnvironmentHttpAccessCommand.php

```diff
@@ -174,7 +174,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
             // Merge existing settings. Not using a reference here, as that
             // would affect the comparison with $current later.
             foreach ($current as $key => $value) {
-                if (!isset($accessOpts[$key])) {
+                // Use null to delete an existing value.
+                if ($accessOpts[$key] === [] && is_array($value) && $value !== []) {
```

Comment:
Aren't you going to potentially get a warning  if you don't do the if !isset() first?

Reply:
true